addons/im_livechat/static/src/embed/common/livechat_button.js

```diff
@@ -43,6 +43,15 @@ export class LivechatButton extends Component {
             },
         });
         useExternalListener(document.body, "scroll", this._onScroll, { capture: true });
+        useEffect(
+            () => {
+                const rootNodeClassList = this.ref.el.getRootNode().host?.classList;
+                this.isShown && rootNodeClassList
```

Comment:
Not found of side effects in ternary. Also either `rootNodeClassList` is never falsy (as show by `!!document.createElement("div").classList`) and the condition should not take it into account, or it is and the second expression in the ternary is subject to crashes (since `rootNodeClassList.remove` would be undefined)

Reply:
No, but the ref can be undefined somehow.